Packs/Base/TestPlaybooks/playbook-TestCommonPython.yml

```diff
@@ -0,0 +1,563 @@
+id: TestCommonPython
```

Comment:
shouldn't this one be a move file? I can see any the addition to the pack but not the removal from the general test playbook dir.